styles.css

```diff
@@ -0,0 +1,43 @@
+html {font-family: 'Poppins', sans-serif;
+
+}
+a 
+{
+  color: black
+}
+body {
+    background-color: #3845;
+    color: rgba(0, 0, 0, 0.445)
```

Comment:
Your CSS uses:
- color keywords
- hex colors
- rgba colors

I recommend sticking to either one format of color values or one format + keywords. See [CSS best practices](https://mica-web.github.io/learn/css/best-practices#use-color-values-logically-and-consistently).

Reply:
Additionally, in order to meet [color contrast recommendations](https://contrastchecker.com/?c=68786b&b=bbd7c0), you would need to increase the size of your font to use this combination (or you can make the body text darker; probably more like `rgba(0, 0, 0, 0.6)`).